examples/IRB140/HandInput.m

```diff
@@ -0,0 +1,16 @@
+classdef HandInput < SingletonCoordinateFrame
+  % hand input coordinate frame
+  methods
+    function obj=HandInput(r, ind, name)
+      typecheck(r,'TimeSteppingRigidBodyManipulator');
```

Comment:
Going with {TSRBM,RBM} to keep the frames more general-purpose. However, these frames are duplicated between Atlas/atlasframes and here. It would be kind of nice to maintain the hand somewhere else where both of those examples can get at it -- would it make sense to maybe move the hand over into its own example folder, and addPath that folder in both the Atlas and IRB140 example cases?